index.html

```diff
@@ -74,5 +74,7 @@ <h2>Hands-Down!</h2>
         </div>
       </div>
     </div>
+
+    <script src="/reload/reload.js"></script>
```

Comment:
Oh this is cool! So when you import a package with npm, its source becomes available at the root directory to all scripts the runtime starts?

Reply:
Yeah, it's an interesting thing about node_modules (I don't know how that works tho).